src/System.Management.Automation/security/wldpNativeMethods.cs

```diff
@@ -64,7 +64,7 @@ public static SystemEnforcementMode GetSystemLockdownPolicy()
 
         private static object s_systemLockdownPolicyLock = new Object();
         private static Nullable<SystemEnforcementMode> s_systemLockdownPolicy = null;
-        private static bool s_wasSystemPolicyDebugPolicy = false;
+        private static bool s_allowDebugOverridePolicy = false;
```

Comment:
I feel that this name is no less confusing.  This indicates that the system wide debug policy is in effect.  I feel a better name would be something like: `s_isDebugSystemPolicy`

Reply:
`s_wasSystemPolicyDebugPolicy` will be set to `true` as long as we can reach the `GetDebugLockdownPolicy(string)` method, and it doesn't matter whether or not the debug policy is set and enforced.

That means, as long as the system lock-down policy is not `Enforce`, it's allowed to use debug override policy, and thus we will check for the debug policy every time `GetSystemLockdownPolicy()` is called.